### PR TITLE
feat: provide tensorflow 2.5 image [DET-5522]

### PR DESCRIPTION
**Pull request by ioga**
## Description

Add Tensorflow 2.5 & CUDA 11.2 image.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -97,11 +97,11 @@ commands:
       - when:
           condition: <<parameters.tf1>>
           steps:
-            - run: docker pull determinedai/environments:py-3.7-pytorch-1.7-tf-1.15-cpu-606fd02
+            - run: docker pull determinedai/environments:py-3.7-pytorch-1.7-tf-1.15-cpu-da845fc
```
Comment (mackrorysd) [inline .circleci/config.yml:100]: Looks like this actually includes @shiyuann 's latest changes, as of "add a github template". Looks like there was a change that got reverted to. Let's have him confirm this is intended for release already.

Comment (shiyuann) [reply .circleci/config.yml:100]: I think `c1b75818ca76628196c5939f80a6ed4912247db1` and `da845fc89f8ff4b03e2018e407ebf5ff545b3067` should both work.

Comment (shiyuann) [reply .circleci/config.yml:100]: `254d51127cc540b30a2016081e3e5b4bffa8d292` work too

Comment (ioga) [reply .circleci/config.yml:100]: his change wasn't intended for that release, but it overwrote 0.15.0, so we've reverted it to overwrite 0.15.0 back, and added a harmless template. so all is correct in my opinion: last material change what the one with horovod cpu.

Comment (ioga) [reply .circleci/config.yml:100]: I want to use the latest so the "hashed" and "versioned" are the same image.

**Review by mackrorysd — APPROVED** [review]

**Comment by ioga** [comment]
(admin-merged because this was an upstream branch, and not all tests were requested)

**Pull request merged**